src/config/limits.js

```diff
@@ -1,3 +1,4 @@
+//update based on therapy settings
 var mgdL = {
```

Comment:
FYI - There are tickets coming through that will make the therapy setting warnings and limits different based upon the age (which is calculated from entered birthdate). Something to think about before it happense.

Reply:
yeah - I am mulling over this in general and was going to have a chat to you. I need dynamic setup based on the therapy settings